tartiflette/sdl/transformers/location.py

```diff
@@ -0,0 +1,49 @@
+from lark.lexer import Token
+from typing import Optional
+
+
+class Location(object):
+
+    __slots__ = [
+        'line',
+        'column',
+        'position_in_stream',
+        'context',
+    ]
+
+    def __init__(
+        self,
+        line: int,
+        column: int,
+        position_in_stream: int,
+        context: str = None
+    ):
+        self.line = line
+        self.column = column
+        self.position_in_stream = position_in_stream
+        self.context = context
+
+    def __eq__(self, other: 'Location'):
+        return isinstance(other, Location) and \
+               self.line == other.line and self.column == other.column \
+               and self.position_in_stream == other.position_in_stream
+
+    def __repr__(self):
+        return 'Location(line: {}, column: {}, ' \
+               'position_in_stream: {}, context: ..."{}"...)'.format(
+                self.line, self.column, self.position_in_stream, self.context)
+
+    @staticmethod
+    def from_token(
+        token: Token, input_sdl: Optional[str] = None, window_size: int = 20
```

Comment:
`input_` prefix is useless

Reply:
Could you add a small description with a sample?

Reply:
It's been deleted